kytos/core/interface.py

```diff
@@ -242,6 +242,12 @@ def default_special_tags(self) -> list[str]:
          special_available_tags and special_tags."""
         return ["untagged", "any"]
 
+    def all_tags_available(self) -> bool:
+        """Return True if all tags are avaiable (no tags used),
+         False otherwise"""
+        with self._tag_lock:
+            return self.available_tags == self.tag_ranges
```

Comment:
To potentially avoid special tags being leaked or left behind when checking this, should also `self.special_available_tags == self.special_tags` be included in this conditional?

Reply:
You are right, forgot about them.